exec/cnex/lib/process_win32.c

```diff
@@ -9,6 +9,29 @@
 #include "nstring.h"
 
 
+struct ThreadInfo {
+    HANDLE pHandle;
+    Cell *buffer;
+};
+
+static DWORD WINAPI reader_thread(LPVOID lpParam)
+{
+    struct ThreadInfo *ti = lpParam;
+
+    for(;;) {
+        char buf[1024];
+        DWORD n;
+        if (ti->pHandle != INVALID_HANDLE_VALUE) {
```

Comment:
You still don't have to check check for an invalid handle here! Nothing is going to suddenly make your handle invalid.

Reply:
Fixed.